tailwind.config.js

```diff
@@ -1,7 +1,11 @@
 /** @type {import('tailwindcss').Config} */
 export default {
   content: ['./lib/**/*.{js,ts,jsx,tsx}'],
+  darkMode: 'selector',
   theme: {
+    screens: {
+      lg: '1440px',
+    },
```

Comment:
Not so familiar with Tailwind, so why there is need to specify just one screen size and specifially this?

Reply:
We should have breakpoints at least for desktop and mobile. Current `1440px ` is from design and it's the only one we know of so far. See default breakpoints https://tailwindcss.com/docs/screens.